.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liquid.format.enable": false
```

Comment:
Is committing this file intentional?  I don't use vscode so I'm not sure how it's normally handled.  i.e. are settings project-specific or person-specific?

Reply:
No, I installed a liquid extension that by default messes up the formatting of the file so I disabled it, but that created a settings json in the project folder. I found the option to disable it globally so the project-specific settings json is not needed.